state/azure/tablestorage/tablestorage.go

```diff
@@ -0,0 +1,206 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package tablestorage
+
+import (
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/dapr/components-contrib/state"
+	jsoniter "github.com/json-iterator/go"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"strings"
+)
+
+const (
+	fullmetadata        = "application/json;odata=fullmetadata"
+	daprDelim           = "__delim__"
+	valueEntityProperty = "Value"
+
+	accountNameKey = "accountName"
+	accountKeyKey  = "accountKey"
+	tableNameKey   = "tableName"
+)
+
+type StateStore struct {
+	table *storage.Table
+	json  jsoniter.API
+}
+
+type tablesMetadata struct {
+	accountName string
+	accountKey  string
+	tableName   string
+}
+
+// ---- Store interface begin
+
+//initialises connection to table storage, optionally creates a table if it doesn't exist
```

Comment:
Nit: Please follow the comment convention - https://golang.org/doc/effective_go.html#commentary

Reply:
fixed, thank you